services/ui-src/src/utils/s3Uploader.js

```diff
@@ -43,19 +45,17 @@ export async function uploadFiles(fileArray) {
           resolve(results);
         })
         .catch((error) => {
-          if (error.indexOf("No credentials") !== -1) {
-            reject("SESSION_EXPIRED");
+          if (error.message.indexOf("No credentials") !== -1) {
+            reject(RESPONSE_CODE.SESSION_EXPIRED);
           } else {
-            console.log("Error uploading.", error);
-            reject("UPLOADS_ERROR");
+            reject(RESPONSE_CODE.UPLOADS_ERROR);
```

Comment:
I like how you cleaned up the literals into their respective constant values even though it wasnt directly part of the code you needed to modify!